docs/src/theme/DocSidebar/Desktop/Content/styles.module.css

```
@@ -1,8 +1,66 @@
 @media (min-width: 62.25rem) {
+  .scrollContainer {
```

Stylelint (17.14.0)

[error] Expected class selector ".scrollContainer" to be kebab-case (selector-class-pattern)

(selector-class-pattern)